### DISCUSSION TITLE
The number of unavailable copies of ingester is greater than or equal to 2, and the entire mimir cluster is unavailable.

### DISCUSSION THREAD

**Discussion by czhfe** [grafana/mimir#6399] [Help and support]
#### Describe the bug
I have a mimir cluster with 1 ingester replicas, as long as the number of ingester unavailable replicas is greater than or equal to 2, the whole mimir cluster is unavailable, I don't know what the problem is.

#### To Reproduce

1. Artificially making 2 ingester unavailable

**Comment by dimitarvdimitrov** [comment]
can you provide some more details? Do you start with 1 ingester replica or with 3? What replication factor are you running with?
